userprofile/urls.py

```diff
@@ -12,5 +12,5 @@
         name="privacy_notice"),
     url(r'^accounts/$', views.ProfileView.as_view(),
         name="profile"),
-    url(r'^setlang', views.LanguageView.set_language, name="set_language"),
+    url(r'^setlang', views.set_user_language, name="set_user_language"),
```

Comment:
Could you still change the name to `set-user-language`, so the url name format would match all other url names in the code base.

Reply:
I did, check the last commit.

Reply:
Sorry, github notifications got me on an old page.. :disappointed: